### ISSUE TITLE
Unable to build fptest in Lazarus 1.4.4

### ISSUE THREAD

**Issue by dsfb** [graemeg/fptest#6]
Hi, Graemeg.

I tried to build fptest in Lazarus 1.4.4, but it failed because Lazarus couldn't find the following package:
"fpgui_toolkit". 
I downloaded your project as zip file, and then I tried to open it in Lazarus, but Lazarus shows the error related with "fpgui_toolkit" package dependency.
Please, help me, because I don't know how to find the "fpgui_toolkit" package.

Best Regards,
dsfb.

**Comment by graemeg** [comment]
Where do you see that error - in your own test project?  There are three convenience packages included in FPTest, though you don't need to use Lazarus or *.lpk packages.

 * **fptest.lpk** - primary FPTest code
 * **fptest_lcl.lpk** - uses fptest.lpk and introduces the LCL based GUI Test Runner.
 * **fptest_fpgui.lpk** - uses fptest.lpk and introduces the fpGUI based GUI Test Runner.

There is also a Console Test Runner, but that only needs fptest.lpk, so no extra packages were needed or created for this.

## FPTest Usage
1. Create your unit test project. In terms of Lazarus IDE, that will be the "Simple Program" project type.
2. If you are going to use the Console (Text) Test Runner,then simply associated the **fptest.lpk** with your project and  include _TextTestRunner_ in your uses clause.
3. If you are going to use the GUI Test Runner, then based on what GUI toolkit you are using (LCL or fpGUI), then associated **fptest_lcl.lpk** or **fptest_fpgui.lpk** respectively with your test project. Then add the _GUITestRunner_ in your uses clause.
4. Register your test any way you want.
5. Call _RunRegisteredTests;_ in your program unit.

I primarily use fpGUI, so some of the  examples included with FPTest might link to the **fptest_fpgui.lpk** package. Simply change that package dependency to **fpgui_lcl.lpk** if you are using LCL. I still need to clean up the demos included with FPTest and clearly separate LCL and fpGUI demos.

I hope this helps.

**Comment by dsfb** [comment]
Thanks for your help.

I tried to build your project, by opening the file: "https://github.com/graemeg/fptest/blob/master/project1.lpi" in my machine, with all other files from your repository, "https://github.com/graemeg/fptest/"...

But, while you was seeing my previous message, I found the following links:
http://wiki.freepascal.org/fpGUI
https://github.com/graemeg/fpgui/
, and then I solved this issue with the first link. However, your answer was good to me, because I know how to test my pascal projects with fptest, now.

Best Regards,
dsfb.
